tests/lax_scipy_test.py

```diff
@@ -576,12 +576,8 @@ def test_spectral_dac_eigh(self, linear_size, seed, dtype):
       'linear_size': linear_size, 'seed': seed, 'dtype': dtype}
     for linear_size in linear_sizes
     for seed in seeds
-    for dtype in jtu.dtypes.floating))
+    for dtype in [jnp.float32, ]))
   def test_spectral_dac_svd(self, linear_size, seed, dtype):
```

Comment:
let's use "bisection" instead of "dac" (which is hardcoded to translate to "digital to analog converter" in my brain ;-) )